tests/test_active_surface.py

```diff
@@ -76,6 +76,212 @@ def test_soft_reset(self):
+        msg = b'\xFA\xA0\x23\x00\x00\x00\x00'
+        checksum = b'\x42'
+        for byte in msg:
+            self.assertTrue(self.system.parse(byte))
+        self.assertEqual(self.system.parse(checksum), byte_ack)
+
+    def test_set_IO_pins(self):
+        msg = b'\xFA\x40\x25\x00'
+        checksum = b'\xA0'
+        for byte in msg:
+            self.assertTrue(self.system.parse(byte))
+        self.assertEqual(self.system.parse(checksum), byte_ack)
+
+    def test_set_resolution(self):
+        msg = b'\xFA\x40\x26\x00'
+        checksum = b'\x9F'
+        for byte in msg:
+            self.assertTrue(self.system.parse(byte))
+        self.assertEqual(self.system.parse(checksum), byte_ack)
+
+    def test_reduce_current(self):
+        msg = b'\xFA\x40\x27\x00'
+        checksum = b'\x9E'
+        for byte in msg:
+            self.assertTrue(self.system.parse(byte))
+        self.assertEqual(self.system.parse(checksum), byte_ack)
+
+    def test_set_finite_delay(self):
+        msg = b'\xFA\x40\x28\x00'  # Setting a delay step equal to 0
+        checksum = b'\x9D'
+        for byte in msg:
+            self.assertTrue(self.system.parse(byte))
+        self.assertEqual(self.system.parse(checksum), byte_ack)
+
+    def test_set_infinite_delay(self):
+        msg = b'\xFA\x40\x28\xFF'  # Setting an infinite delay
+        checksum = b'\x9E'
+        for byte in msg:
+            self.assertTrue(self.system.parse(byte))
+        self.assertTrue(self.system.parse(checksum))
+
+    def test_toggle_delayed_execution(self):
+        msg = b'\xFA\x40\x29\x00'
+        checksum = b'\x9C'
+        for byte in msg:
+            self.assertTrue(self.system.parse(byte))
+        self.assertEqual(self.system.parse(checksum), byte_ack)
+
+    def test_set_absolute_position(self):
+        msg = b'\xFA\xA0\x30\x00\x00\x00\x00'
+        checksum = b'\x35'
+        for byte in msg:
+            self.assertTrue(self.system.parse(byte))
+        self.assertEqual(self.system.parse(checksum), byte_ack)
+
+    def test_set_relative_position(self):
+        msg = b'\xFA\xA0\x31\x00\x00\x00\x00'
+        checksum = b'\x34'
+        for byte in msg:
+            self.assertTrue(self.system.parse(byte))
+        self.assertEqual(self.system.parse(checksum), byte_ack)
+
+    def test_rotate(self):
+        msg = b'\xFA\x40\x32\x00'  #Rotate according to velocity. Sign = direction
+        checksum = b'\x93'
+        for byte in msg:
+            self.assertTrue(self.system.parse(byte))
+        self.assertEqual(self.system.parse(checksum), byte_ack)
+
+    def test_set_in_range_velocity(self):
+        msg = b'\xFA\x80\x35\x00\x00\x00'  # Velocity = 0, OK
+        checksum = b'\x50'
+        for byte in msg:
+            self.assertTrue(self.system.parse(byte))
+        self.assertEqual(self.system.parse(checksum), byte_ack)
+
+    def test_set_out_range_velocity(self): # Velocity = 8388607, too high
+        msg = b'\xFA\x80\x35\xEF\xFF\xFF'
```

Comment:
Can you please write the comment in the line just above the `msg = b...`?  In the comment, something like:
`# Velocity=8388608 (\xYY\xZZ...), higher than the max value 8000000.`